tests/Controllers/EmployeeControllerTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\TestCase;
+use App\Services\Eloquent\EmployeeService;
+use Illuminate\Support\Carbon;
+use PHPUnit\Framework\Constraint\Exception;
+
+class EmployeeControllerTest extends TestCase
+{
+    use DatabaseTransactions;
+    private $employeeService;
+
+    private $urlApi = '/api/v1/employees/';
+
+    protected $faker;
+
+    private $mock;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->mock = App::make(EmployeeService::class);
+        $this->employeeService = $this->mock(EmployeeService::class);
+        $this->faker = Faker\Factory::create();
+        $this->initData();
+    }
+
+    public function mock($class)
+    {
+        $mock = \Mockery::mock($class);
+        $this->app->instance($class, $mock);
+        return $mock;
+    }
+
+    public function tearDown(): void
```

Comment:
if this function don't have different with parent class, you don't need override it.